### PR TITLE
test stability of storage key produced by the capabilities resolver (kt)

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt [hunk 1]
```diff
@@ -128,21 +128,28 @@ class CapabilitiesResolverTest {
...
-            DatabaseStorageKey::class.java
-        )
+        val ramdiskKey =
+            resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId)
+        verifyStorageKey(ramdiskKey, RamDiskStorageKey::class.java)
+        assertThat(ramdiskKey).isEqualTo(resolver1.createStorageKey(
+            Capabilities.TiedToRuntime,
+            thingSchema,
+            handleId
+        ))
```
Comment (jasonwyatt) [inline javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: is this required? it's calling the same method on the same object, with the same args as on line 131, so the result should be the same anyway?

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt [hunk 2]
```diff
@@ -128,21 +128,28 @@ class CapabilitiesResolverTest {
...
+
+        val persistentKey =
+            resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId)
+        verifyStorageKey(persistentKey, DatabaseStorageKey::class.java)
+        assertThat(persistentKey)
+            .isEqualTo(resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId))
```
Comment (jasonwyatt) [inline javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: Same here?

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt [hunk 3]
```diff
@@ -128,21 +128,28 @@ class CapabilitiesResolverTest {
...
         val resolver2 = CapabilitiesResolver(options)
-        verifyStorageKey(
-            resolver2.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId),
-            VolatileStorageKey::class.java
-        )
+        val volatileKey =
+            resolver2.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId)
+        verifyStorageKey(volatileKey, VolatileStorageKey::class.java)
+        assertThat(volatileKey)
+            .isEqualTo(resolver2.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId))
```
Comment (jasonwyatt) [inline javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: And here..

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt (hunk 1 above)
Comment (jasonwyatt) [reply javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: If you want to keep it, that's :+1: , but the styling should look like this:

```kotlin
        assertThat(ramdiskKey).isEqualTo(
            resolver1.createStorageKey(
                Capabilities.TiedToRuntime,
                thingSchema,
                handleId
            )
        )
```

Comment (mmandlis) [reply javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: dunno, this isn't the brightest test ever, but it makes sure that random unique id generation not going on in CapabilitiesResolver...

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt (hunk 2 above)
Comment (mmandlis) [reply javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: done. 
kotlin is a language optimized for people with veeeeery narrow screens!

File: javatests/arcs/core/storage/CapabilitiesResolverTest.kt (hunk 3 above)
Comment (mmandlis) [reply javatests/arcs/core/storage/CapabilitiesResolverTest.kt]: done

**Pull request merged**